JitsiConference.js

```diff
@@ -1109,6 +1119,17 @@ JitsiConference.prototype._setupNewTrack = function(newTrack) {
         this.room.setVideoMute(newTrack.isMuted());
     }
 
+    // Setup E2EE on the new track that has been added
```

Comment:
@jallamsetty1 Why not to put that code into _doReplaceTrack ? There is a code which also handles p2p & jvb track replace. So I think it could be a better place to setup sender.
Also, _setupNewTrack before that, did not depend on p2p/jvb jingle sessions.

Reply:
To me this constitutes as an action that needs to be performed while handling the addition of the new track at the conference level rather than the replace operation. So it makes more sense to add it here.